package.json

```diff
@@ -52,7 +52,7 @@
     "test": "c8 mocha build/test/*.js",
     "predocs-test": "npm run docs",
     "docs-test": "linkinator docs",
-    "samples-test": "c8 mocha samples/test/*.js",
+    "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
```

Comment:
`cd ./` shouldn't be required, because `npm` runs inside its own shell:

`cd samples/ && npm link ../ && npm test` (we should update our docs to reflect this if we suggest the `cd ../` anywhere).

Reply:
Ah, okay. I copied this script from nodejs-bigquery, so I'll also update it there.